content/maxwell1_fundamentals/maxwells_equations/maxwells_equations_in_homogeneous_media/frequency_domain/dipole_sources/electric_dipole/analytic_solution.rst

```diff
@@ -8,3 +8,81 @@ Analytic Solution
     Purpose here
 
 
+The electric current dipole is an elementary length of current flowing in a single direction. It is convenient to solve the problem in terms of Schelkunoff potentials as done in :cite:`ward1988`. To summarize, for an electric current source (:math:`\mathbf{J_e^s}`), Maxwell's equations in frequency are
+
+.. math::
+	\nabla \times \mathbf{E_e} + i\omega \mu \mathbf{H_e} = 0 
+	:label: Faraday_e
+.. math::
+	\nabla \times \mathbf{H_e} - (\sigma + i\omega \epsilon) \mathbf{E_e} = \mathbf{J}_e^s 
+	:label: Ampere_e
+
+where the subscripts :math:`_e` remind us that we are using an electric source. Taking the curl of :eq:`Faraday_e` allows us to define :math:`\mathbf{H_e}` as the curl of a vector potential (:math:`\mathbf{A}`). 
```

Comment:
@dccowan: should not it be "Taking the divergence of" instead of the "Taking the curl"?